egyed-janosne.jade

```diff
@@ -41,6 +41,9 @@ block gallery
     a.pretty-photo.zoom(href='images/muveszeink/egyed-janosne/dana-ekszer-8.jpg', title='Dana - Ékszer')
       img.img-border(src='images/muveszeink/egyed-janosne/dana-ekszer-8-383x233.jpg', alt='Dana - Ékszer', style='width:383px;height:233px')
   .gallery-item.type-gallery-item.gallery-item.isotope-item
+    a.pretty-photo.zoom(href='images/muveszeink/egyed-janosne/dana-ekszer-9-(2).jpg', title='Dana - Ékszer')
+      img.img-border(src='images/muveszeink/egyed-janosne/dana-ekszer-9-(2).jpg', alt='Dana - Ékszer', style='width:383px;height:233px')
```

Comment:
Can I ask you please to replace the thumbnail to the smaller picture: `dana-ekszer-9-(2)-383x233.jpg`